roles/engineblock/defaults/main.yml

```diff
@@ -38,6 +38,10 @@ engine_debug: false
 
 engine_profile_baseurl: "https://profile.{{ base_domain }}"
 
+# Required for Stepup authentication
+engine_stepup_base_domain: "stepup.example.com"
```

Comment:
Could `example.com` also instead be `{{ base_domain }}`?

Reply:
Done